### ISSUE TITLE
GCAM 5.1.2 issue when using the "pasture_ssp34.xml" file

### ISSUE THREAD

**Issue by IanBCG** [JGCRI/gcam-core#44]
I'm doing some work with the SSPs. I recently switched over to using GCAM 5.1.2, and since I made the switch I've had trouble conducting runs using the "pasture_ssp34.xml" file. If I include the file in my run, the run crashes while solving the 1975 period. The error message reads: 

"The sum of land areas in region Africa_Eastern exceeds the land allocation by 2117.26 (36.3838%)  in period 0"

I didn't experience this problem when using GCAM 5.1.1. Does anyone have guidance on 1) how to resolve the issue or 2) what effect excluding "pasture_ssp34.xml" has on a scenario?

Thanks for your help!

**Comment by RawisWar** [comment]
Hi Ian, 

I got the similar error message when I tried to run the "batch_SSP_SPA1.xml" file on the version 5.1.2.
Please see the attached file. 
![capture](https://user-images.githubusercontent.com/34580273/47530732-6ad84180-d879-11e8-8d8a-5a907fd8591a.PNG)

**Comment by kvcalvin** [comment]
@RawisWar and @IanBCG: Can you remove all of the add on files in your base configuration file before running with the SSPs? I suspect that those files are conflicting with what is in the batch_SSP files.

**Comment by davidrmorrow** [comment]
I've encountered the same error when running SSP4m in 5.1.2: "sum of land areas in region Africa_Eastern exceeds the land allocation by 2117.27 (36.3838%) in period 0."

@pralitp & @kvcalvin: I'm running a new install of 5.1.2 on Windows 10. If I start with the default configuration.xml and set the batch file to batch_SSP_REF.xml, it throws the error about Africa_Eastern during Period 0. If I delete the SSP4 FileSet, it runs SSPs 1, 2, and 5 without incident and then throws the same error when it gets to SSP3.

So far I've tried the following, all with the same result:
- Removing the GCAM ComponentSet from batch_SSP_REF.xml (same error)
- Removing all ScenarioComponents from configuration.xml and using only the scenario components in batch_SSP_REF.xml (same error)
- Adding pasture_ssp34.xml as a ScenarioComponent to the default configuration.xml file, without any of the other SSP files (same error)

I've also tried the following, which gives me the corresponding error about land allocation in region USA and the several hundred errors about being unable to "delete node biomass," as shown in @RawisWar's post:
- Default configuration.xml with batch_SSP_REF.xml with pasture_ssp34.xml commented out.
- Default configuration.xml with batch_SSP_REF.xml with all of the AGLU files commented out.

I'm attaching the comparison between pasture_ssp34.xml in 5.1.1 and 5.1.2: ![pasture_ssp34_comparison](https://user-images.githubusercontent.com/24574551/49230244-377b5c00-f3bd-11e8-9a8d-a9ab4d5dade7.PNG)

**Comment by kvcalvin** [comment]
Hi @davidrmorrow,

The errors you are seeing are with your first, third, fourth, and fifth test are expected. Essentially, the `pasture_ssp34.xml` file is incompatible with the protected_land files, so both cannot be read into the model at the same time. The SSPs were designed to work without protected lands. So, the proper configuration should include nothing in the <ScenarioComponents> section of the configuration file and using the `batch_SSP_REF.xml` as is. Can you confirm that is what you tried with your second option?

Thanks!

**Comment by davidrmorrow** [comment]
Thanks, @kvcalvin. I'd _thought_ that's what I'd tried for my second option, but it looks like I made a mistake. Long story short: while investigating the effects of the `protected_land` files during my troubleshooting, I'd accidentally inserted them into the original `batch_SSP_REF.xml` file instead of my modified one, so the second option described above still included the protected land files. I've now reverted to the default `batch_SSP_REF.xml` file and cleared out all the ScenarioComponents from `configuration.xml`, after which I was able to run the SSPs successfully. Sorry to have bothered you with this when I should have been able to figure it out on my own!
